src/GopayConfig.php

```diff
@@ -14,7 +14,13 @@ class GopayConfig
 	/**
 	 * Parametr specifikujici, pracuje-li se na testovacim ci provoznim prostredi
 	 */
-	static $version = self::TEST;
+	public static $version = self::TEST;
+
+	/**
+	 * @var callable
+	 */
+	public static $soapClientFactory = '\Markette\Gopay\Api\SoapClientFactory::create';
```

Comment:
Almost everything in API is static.

So I've added static property with factory.

The default one simply creates `new \SoapClient($wsdl, array())` which was created on many places before.